packages/bpk-component-nudger/README.md

```diff
@@ -47,19 +49,104 @@ class App extends Component {
 }
 ```
 
+### Configurable `BpkConfigurableNudger`
+
+```js
+import React, { Component } from 'react';
+import BpkLabel from 'bpk-component-label';
+import { BpkConfigurableNudger } from 'bpk-component-nudger';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      value: 1,
+    };
+  }
+
+  handleChange = value => {
+    this.setState({ value });
+  };
+
+  compareValues = (a, b): number => {
+    const options = ['economy', 'premium', 'business', 'first'];
+    const [aIndex, bIndex] = [options.indexOf(a), options.indexOf(b)];
+    return aIndex - bIndex;
+  };
+
+  incrementValue = (a): string => {
```

Comment:
Can we make this a bit more explicit? We should aim to avoid variables with undescriptive names

```suggestion
  incrementValue = (currentValue): string => {
```